### ISSUE TITLE
Error when running v0.0.10 on Win10

### ISSUE THREAD

**Issue by rejas** [yuanqing/gulp-tape#23]
After upgrading gulp-tape from v0.0.9 to v0.0.10 I get this error:

```
events.js:183
      throw er; // Unhandled 'error' event
      ^
Error: Command failed: C:\Projekte\swisscom\swidch-ui-light\frontend\node_modules\tape\bin\tape
Der Befehl "C:\Projekte\ui-light\node_modules\tape\bin\tape" ist entweder falsch geschrieben oder konnte nicht gefunden werden.

    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
error An unexpected error occurred: "Command failed.
Exit code: 1
```

Sorry for the german error message in there, it basically says: The command is either misspelled or could not be found. 
I tend to blame my windows10 for it, because on the (linux) servers the update didnt start to fail.

**Comment by yuanqing** [comment]
It seems from the error message that the `tape` binary could not be found. Is the `tape` module installed? The cause of the error might be https://github.com/yuanqing/gulp-tape/blob/7754fb0e2956feac10352c4d3d10fa305d2c96f1/index.js#L8

**Comment by rejas** [comment]
The file is there @yuanqing, the problem I think is that it doesnt have a file extension (js in this case) and windows doesnt understand the shebang line 
`#!/usr/bin/env node`
at the beginning (as opposed to linux who could figure out what to do with it)

I rather think you need to point the filepath to the "node_moules/bin/tape" file which gets created by npm/yarn during install so that OS specifics are handled. Maybe I can provide a PR if I find the time to test this hypothesis...

**Comment by yuanqing** [comment]
Maybe we could try to use a module like https://www.npmjs.com/package/resolve-bin to get the path to the `tape` binary

**Comment by rejas** [comment]
Could be possible. Then maybe the tests would run through :-) 

But before I have to ask what the reasoning is behind calling tape via childProcess.exec and not just require it and call it?

**Comment by yuanqing** [comment]
Running the tests via `require` was [the approach taken in `0.0.9` and before](https://github.com/yuanqing/gulp-tape/blob/eac096b82ce4b22832c97c6f6e30fb83860f9366/index.js). This approach involved using `tape` module internals, and introduced complications with `gulp.watch` (https://github.com/yuanqing/gulp-tape/issues/13)

Do check if `resolve-bin` fixes your issue on Windows; happy to review and merge in your PR

**Comment by rejas** [comment]
Tried running it with the resolve-bin but that doesnt fix it. resolve-bin will also point to the js file in the tape-bin folder, but what is needed is the tape-cmd in the node_modules-bin-folder

**Comment by yuanqing** [comment]
@rejas Understand now. Afraid I don’t have a Windows machine, so I cannot test.

What is the name of the `tape` binary in `./node_modules/.bin/`? Does changing the `require.resolve` call to `require.resolve('.bin/tape')` work?

```js
const TAPE_BINARY_FILEPATH = require.resolve('.bin/tape')
```

(On Unix, `./node_modules/.bin/tape` is a symlink to `./node_modules/tape/bin/tape`.)

**Comment by ghost** [comment]
I have a Windows machine, and `require.resolve('.bin/tape')` seems to do the trick.

**Comment by rejas** [comment]
Indeed, it works for me too. I updated my PR for this issue accordingly.

**Comment by yuanqing** [comment]
Thanks @rejas and @TheDancingCode for checking! I’ve merged the PR and will cut a `0.0.11` shortly
